### PR TITLE
Improvements to PowerShell install script

### PR DESCRIPTION
**Pull request by artemmnad**
Proposed changes to the install script:
- Make it more readable and easily maintainable thanks to named parameters;
Valid commands: `.\install.ps1 -v v0.2.0`, `.\install.ps1 -version v0.2.0`, `.\install.ps1 v0.2.0`
- Make possible the usage following command
`$install = iwr https://deno.land/x/install/install.ps1; iex "$install v0.2.0"`
This way the `install.ps1` file is not created needlessly after download.
This approach would not be possible with the previous implementation (correct me if I'm wrong).

No need to change code in CI. https://github.com/denoland/deno_std/pull/89

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,5 @@ curl -L https://deno.land/x/install/install.py | python - v0.2.0
 
 **Install with PowerShell:**
 
-```
-iwr https://deno.land/x/install/install.ps1 -out install.ps1; .\install.ps1 v0.2.0
-```
+```powershell
+$install = iwr https://deno.land/x/install/install.ps1; iex "$install v0.2.0"
```
Comment (MarkTiedemann) [inline README.md]: This doesn't work, unfortunately.

```
PS C:\> $install = iwr https://raw.githubusercontent.com/artemmnad/deno_install/feature/powershell-specify-version/
install.ps1
PS C:\Users\wwwma> iex "$install v0.2.0"
Downloading https://api.github.com/repos/denoland/deno/releases/latest... done.
Determining latest Deno release: v0.2.5...  done.
Downloading https://github.com/denoland/deno/releases/download/v0.2.5/deno_win_x64.zip... done.
Extracting C:\Users\<redacted>\.deno\bin\deno_win_x64.zip into C:\Users\<redacted>\.deno\bin... done.
Removing C:\Users\<redacted>\.deno\bin\deno_win_x64.zip... done.

Deno was installed successfully.
Run deno --help to get started.

iex : The term 'v0.2.0' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ iex "$install v0.2.0"
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (v0.2.0:String) [Invoke-Expression], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.InvokeExpressionCommand
```

Comment (MarkTiedemann) [reply README.md]: How about this?

```powershell
icm ([scriptblock]::Create("$(iwr $url)")) -Args v0.2.0
```

Comment (artemmnad) [reply README.md]: Looks great but you're right. 
It is significantly less readable and I think that readability is more important than not creating a temp file.

File: install.ps1 [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Copyright 2018 the Deno authors. All rights reserved. MIT license.
 # TODO(everyone): Keep this script simple and easily auditable.
+param
```
Comment (MarkTiedemann) [inline install.ps1]: I think the formatting here could be improved a tiny bit.

```suggestion
param (
  [alias("v")]
  [string] $version
)
```

File: README.md [hunk 3]
```diff
@@ -32,6 +32,6 @@ curl -L https://deno.land/x/install/install.py | python - v0.2.0
 
 **Install with PowerShell:**
 
-```
+```powershell
```
Comment (MarkTiedemann) [inline README.md]: Good catch!

**Comment by MarkTiedemann** [comment]
LGTM (but I don't have write access). @ry?

**Pull request merged**